base/decfp.jl

```diff
@@ -0,0 +1,32 @@
+# This file is a part of Julia. License is MIT: http://julialang.org/license
+
+# These define the types for IEEE 754-2008 Decimal Floating Point formats
+# Support for these types is available via packages, such as DecFP.jl
+
+export AbstractFlt, BinaryFloat, DecimalFloat, DecimalFloatBinary, DecimalFloatPacked
+
+"""Abstract types for all floating point types"""
+abstract AbstractFlt{b} <: Real
```

Comment:
Please, no pun with `AbstractFlt` vs. `AbstractFloat`. How are we supposed to find out the difference?